Dockerfile

```diff
@@ -11,7 +11,7 @@ LABEL maintainer Netlify
 ENV LANGUAGE en_US:en
 ENV LANG en_US.UTF-8
 ENV LC_ALL en_US.UTF-8
-ENV PANDOC_VERSION 2.4
+ENV PANDOC_VERSION 2.12
```

Comment:
the latest version seems to be 2.13 (https://github.com/jgm/pandoc/releases/tag/2.13). Is there a reason for not updating to this one?